include/alpaka/acc/AccGpuHipRt.hpp

```diff
@@ -0,0 +1,312 @@
+/**
+* \file
+* Copyright 2014-2016 Benjamin Worpitz, Rene Widera
+*
+* This file is part of alpaka.
+*
+* alpaka is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* alpaka is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with alpaka.
+* If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#ifdef ALPAKA_ACC_GPU_HIP_ENABLED
+
```

Comment:
Can we have some check here that the compiler really supports HIP? I had seen something some time ago on one of the branches.